scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.js

```diff
@@ -66,31 +67,11 @@ export function ContentProfileSchemaEditor(content, config) {
 
             scope.formattingOptionsEditor3 = [
                 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'ordered list', 'unordered list', 'quote', 'media', 'link',
-                'embed', 'underline', 'italic', 'bold', 'table', 'formatting marks'
+                'embed', 'underline', 'italic', 'bold', 'table', 'formatting marks', 'annotation', 'comments',
+                'suggestions'
             ];
 
-            if (config.features.editorHighlights) {
```

Comment:
It was a requirement to be able to enable/disable highlights functionality from config file. Why did you dropped this feature?